receiver/windowsperfcountersreceiver/internal/pdh/pdh_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build windows
+
+package pdh
```

Comment:
Do we currently run tests on Windows for this repo?

Reply:
Good question - it doesn't look like we do. This code is copied from Core so I can promise it succeeds 😄 

I'll create a followup PR to add a **windows-build** step for Contrib, similar to what @bogdandrutu previously added for Core.

Reply:
👍 SGTM.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1228